jni/lib/src/jprimitives.dart

```diff
@@ -6,92 +6,92 @@
 // lowercase.
 // ignore_for_file: camel_case_types
 
-import 'types.dart';
+part of 'types.dart';
```

Comment:
I assume this is needed because the whole class hierarchy needs to be in the same "file"?

Reply:
Yes! Because of `sealed`.